src/hdcontroller.js

```diff
@@ -352,7 +352,7 @@ class HDProxydClient {
                     done(err);
                 }
             }, {}, payload);
-        }, err => {
+        }, (err) => {
```

Comment:
Could be simplified with a simple call to the callback, as there is no logic here (same for other places + on the TS file of course)